src/components/TransactionFlow/MultiTxReceipt.tsx

```diff
@@ -99,7 +100,17 @@ export default function MultiTxReceipt({
         const txUrl = buildTxUrl(network.blockExplorer, transaction.txHash!);
 
         const assetRate = getAssetRate(asset);
-
+        const recipient = receiverAddress || to!
+        const valueTransfer = {
```

Comment:
Utility function?

Reply:
Changed the name to `transferEvent` inside of receipts to indicate difference between a valueTransfer and it's receipt-viewable transfer event.